package.json

```diff
@@ -0,0 +1,89 @@
+{
+	"name": "beyond-yoga-snapfu",
+	"author": "Jongsoo Yoon",
+	"version": "0.1.0",
+	"license": "MIT",
+	"main": "index.js",
+	"private": true,
+	"scripts": {
+		"analyze": "webpack --config webpack.modern.js --analyze",
+		"analyze:universal": "webpack --config webpack.universal.js --analyze",
+		"build": "rm -rf ./dist && webpack --config webpack.modern.js && webpack --config webpack.universal.js",
+		"cypress": "cypress open --project tests",
+		"clean": "rm -rf ./node_modules && rm -f ./package-lock.json",
+		"dev": "kill $(lsof -ti:3333,8888); webpack serve --config webpack.dev.js",
+		"format": "prettier --write \"**/*.{js,jsx,ts,tsx,json}\"",
+		"lint": "eslint --fix \"./src/**/*.{js,jsx,ts,tsx}\"",
+		"test": "kill $(lsof -ti:3333,8888); webpack serve --config webpack.dev.js & ((cypress run --project tests || cypress run --project tests) && kill $(lsof -ti:3333,8888))",
+		"lighthouse": "lhci --config ./tests/lighthouse/lighthouserc.js autorun",
+		"prepare": "husky install"
+	},
+	"searchspring": {
+		"siteId": {
+			"t02200": {
+				"name": "beyond-yoga.com"
```

Comment:
The name here needs to match what is found in the SMC. You need to use: `beyondyoga.com`